src/cpp/fastdds/xtypes/type_representation/TypeIdentifierWithSizeHashSpecialization.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
```suggestion
// Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Reply:
Updated in [cfa3f78](https://github.com/eProsima/Fast-DDS/pull/4177/commits/cfa3f781c09d678528b673acc9feef170c546bc0)